src/libs/ajax/Metrics.ts

```diff
@@ -6,21 +6,21 @@ import { getConfig } from 'src/libs/config';
 import { withErrorIgnoring } from 'src/libs/error';
 import { MetricsEventName } from 'src/libs/events';
 import * as Nav from 'src/libs/nav';
-import { authStore, getSessionId } from 'src/libs/state';
+import { AuthState, authStore, getSessionId } from 'src/libs/state';
 import { v4 as uuid } from 'uuid';
 
 export const Metrics = (signal?: AbortSignal) => {
   const captureEventFn = async (event: MetricsEventName, details: Record<string, any> = {}): Promise<void> => {
     await ensureAuthSettled();
-    const { signInStatus, registrationStatus } = authStore.get(); // NOTE: This is intentionally read after ensureAuthSettled
-    const isRegistered = signInStatus === 'signedIn' && registrationStatus === 'registered';
```

Comment:
A registered user should not be identified before signing in. The data in the authStore should not be populated before the user signs in. So in this case, this check is redundant, since you could only be registered in the auth store if you already signed in